src/App/components/Pagination.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Pagination, PaginationItem, PaginationLink } from 'reactstrap';
+
+const PaginationModule = (props) => {
+    return (
```

Comment:
This pagination will do the work for you. Just render the page nos with a for loop